packages/govuk-frontend/src/govuk/helpers/_typography.scss

```diff
@@ -114,19 +139,34 @@
 ///
 /// @access public
 
-@mixin govuk-typography-responsive($size, $override-line-height: false, $important: false) {
+@mixin govuk-font-size($size, $line-height: false, $important: false, $override-line-height: false) {
   @if not map-has-key($govuk-typography-scale, $size) {
     @error "Unknown font size `#{$size}` - expected a point from the typography scale.";
   }
 
+  @if $override-line-height != false {
```

Comment:
I don't think we need to do this, as long as we make it clear what's changed in the release notes. We can change the mixin name and the name of the params at the same time.

As long as the `govuk-typography-responsive` mixin keeps the same params as before, this won't break anything.